tests/integration/test_batches.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from labelbox import Project, Dataset
+
+
+def test_create_batches(project: Project, big_dataset: Dataset):
+    data_rows = [dr.uid for dr in list(big_dataset.export_data_rows())]
```

Comment:
to save on an extra api call, can the `big_dataset` fixture just return those data rows?

Reply:
Possibly - this fixture is used by other tests as well so it would depend on what those tests need the dataset for

Reply:
Looks like some tests need more than the data row ids, so inlining wouldn't work for all cases